### ISSUE TITLE
Savegame location problem (new location can't be accessed by the game)

### ISSUE THREAD

**Issue by RealRaptoReX** [VFansss/mgs2-v-s-fix#124]
Description of the bug:
The latest version tries to move the savefiles to the new location. It succeeds in doing so, but the result is, that the game can't read the savefiles from the new location (documents/mygames/METAL GEAR SOLID 2 SUBSTANCE).
As a result, I can't change the fixs settings and save, because then, I have to save, which again moves the savefiles. If i have savefile in both locations, i have to delete one, because otherwise the fix closes when you go into the settings (saying there are 2 locations with savefiles).
TLDR: The game can't read savefiles from "documents/mygames/METAL GEAR SOLID 2 SUBSTANCE"

Information on my machine:
- OS: Windows 10,
- VGA Model: Nvidia GeForce GTX 1070Ti (MSi)

Log file:
[MGS2_VFix_debug.txt](https://github.com/VFansss/mgs2-v-s-fix/files/5023780/MGS2_VFix_debug.txt)

**Comment by LobizonXP** [comment]
VGA Model is that NVIDIA GeForce GTX 1070 Ti that you have inside your computer.

**Comment by RealRaptoReX** [comment]
> VGA Model is that NVIDIA GeForce GTX 1070 Ti that you have inside your computer.

Haha thanks, stating as graphics unit or graphics card wouldn't lead to this confusion xD

**Comment by VFansss** [comment]
I've updated the wiki.

Probably a re-install and [THIS](https://github.com/VFansss/mgs2-v-s-fix/wiki/Downloading-&-Installing#-unzipping-the-fix) would solve the problem
